docs/adr-0004-versioning.md

```diff
@@ -0,0 +1,75 @@
+# ADR 4: Versioning 5/1/2019
+## Context
+We build several different versions of Firefox for purposes such as release, beta testing, and staged rollout. It's not obvious how the version numbers are related to these types of builds, and features have sometimes disappeared in rollback versions, so we document them here to avoid confusion.
+
+Firefox TV releases are listed [https://github.com/mozilla-mobile/firefox-tv/releases](here).
+
+As of time of writing, the current release version is `3.8`.
+
+## Decision
+Firefox TV versioning is based off of [https://semver.org/](semantic versioning) of MAJOR.MINOR.PATCH, but reflects features rather than API compatibility.
+
+Additionally, we also use alphanumeric suffixes to clearly differentiate between early test builds, releases, and re-releases.
+
+Each release has a *tag* prefixed by `v`, such as `v3.8` and are listed in the [https://github.com/mozilla-mobile/firefox-tv/tags](Tags) page of the repo.
+
+### Semantic Versioning
+* MAJOR version changes signal significant changes to UI or functionality
+* MINOR version changes are released every Sprint, unless they are skipped for release blockers
+* PATCH version changes are for critical bug-fixes that cannot wait for the next Sprint.
+* (LETTER-SUFFIX) reflects builds for our additional purposes that are detailed in following sections.
+
+### Release
+As of 3.8, releases have no suffix, and are released using the staged rollout capability of the Amazon Developer portal.
```

Comment:
nit: "public releases" ? To distinguish between LATs, which are technically also releases?